include/openenclave/bits/eeid.h

```diff
@@ -59,6 +59,21 @@ typedef struct _oe_eeid
 } oe_eeid_t;
 OE_PACK_END
 
+OE_PACK_BEGIN
+/**
+ * Structure to keep EEID endorsements.
+ */
+typedef struct
+{
+    /** Size of the underlying SGX endorsements. */
+    size_t sgx_endorsements_size;
+    /** Size of EEID endorsements (EEID data). */
+    size_t eeid_endorsements_size;
```

Comment:
My understanding is that the "eeid_endorsement" is the eeid configuration data. Comparing against the "customer claims", which can be considered "run time" claims (that can be produced by the any enclave code, including code downloaded into the enclave post enclave initialization), for example, a public key to be used to wrap secret to provision to the enclave, the eeid configuration data can be considered as "init time" claim that captures the identity of the code to be downloaded into the enclave post enclave initialization. The eeid configuration data seems to be fit better in the evidence buffer, together with the "customer claims", than in the endorsement buffer, which typically contains signature/cert-chains.

Reply:
There's already a copy in the evidence ([here](https://github.com/wintersteiger/openenclave/blob/ac364a2cbb60bbc4fb5b378dc8d11592652ab095/include/openenclave/internal/eeid.h#L204)). The second copy in the endorsements is just for convenience; the verifier will check equality with the copy in the evidence. This means that users can replace all or parts of the endorsements with their own, which we clearly need for SGX endorsements, so we might as well put the EEID data into the endorsements too, but users can replace/remove it if they don't trust them. It doesn't make a big difference though, we can remove it if you prefer.

Reply:
OK to have two copies, as long as the Verifier plugin also output relevant claims based on the verified copy.